Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM calico/build:latest
```

Comment:
We can probably reduce the size of this container by using a different base image (e.g scratch).  I don't _think_ we need anything present in the `calico/build` container (though happy to be corrected)